splunk_add_on_ucc_modinput_test/common/splunk_instance.py

```diff
@@ -21,11 +21,57 @@
 
 class Configuration:
     @staticmethod
-    def get_index(index_name: str, client_service: SplunkServicePool) -> Index:
+    def get_index_from_classic_instance(
+        index_name: str,
+        client_service: SplunkServicePool,
+        acs_stack: str,
+        acs_server: str,
+        splunk_token: str,
+    ) -> Index:
+        url = f"{acs_server}/{acs_stack}/adminconfig/v2/indexes/{index_name}"
+
+        headers = {
+            "Authorization": "Bearer " + splunk_token,
+            "Content-Type": "application/json",
+        }
+        response = requests.get(url, headers=headers)
+        if response.status_code == 200:
+            if not client_service._host.startswith(acs_stack):
+                new_host_start = client_service._host.find(acs_stack)
```

Comment:
This function is only called when the stack is cloud but if the stack is not classic then `new_host_start` will not be set and there will be `error` thrown. Setting a default value for `new_host_start` would help prevent any potential undefined variable errors and make the code more readable.